### ISSUE TITLE
RakuDist test fails

### ISSUE THREAD

**Issue by melezhik** [css-raku/HTML-Canvas-raku#2]
Hi!

Here is the log - http://rakudist.raku.org/sparky/report/debian/621

```
20:00:11 08/14/2020 [bash: zef install HTML::Canvas] [HTML::Canvas] # Failed test 'MeasureText'
20:00:11 08/14/2020 [bash: zef install HTML::Canvas] [HTML::Canvas] # at t/render-pdf-test-sheets.t line 180
20:00:11 08/14/2020 [bash: zef install HTML::Canvas] [HTML::Canvas] # text measurement 144.00634765625 outside of range: 110...130
20:00:21 08/14/2020 [bash: zef install HTML::Canvas] [HTML::Canvas] # You failed 1 test of 13
```
HTH

Aleksei

**Comment by dwarring** [comment]
Thanks Aleksie,
Failing test is somewhat System/Font dependent. 

Fixed in HTML::Canvas 0.10.0, uploaded to CPAN.

Clean build http://rakudist.raku.org/sparky/report/debian/631

**Comment by melezhik** [comment]
cool
